README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
**Step 4 lab - Leave a review that asks for changes**
Seriously, this is just a readme file, what to change about?